### DISCUSSION TITLE
WebOS LG app

### DISCUSSION THREAD

**Discussion by muhammedAlySawalhy** [pavjacko/renative#940] [Q&A]
Hello please I want to start the app when the os starts.mean WebOS

**Comment by aakashcdn** [comment]
rnv run -p webos this command works.

**Reply by muhammedAlySawalhy** [reply]
No,sir I mean that the app starts everytime the os boot complete

**Reply by aakashcdn** [reply]
webos simulator automatically run when we change our code.

**Comment by pauliusguzas** [comment]
@muhammedAlySawalhy thanks for using renative, I don't this this issue is in renative scope, this can be done in os proccesses, you can add proccesses on startup, so you can make that `rnv run -p webos` is ran everytime the os boots
